Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -263,6 +263,14 @@ DECLARE_SOA_EXPRESSION_COLUMN(DetectorMap, detectorMap, uint8_t, //! Detector ma
                                 ifnode(aod::track::trdPattern > (uint8_t)0, static_cast<uint8_t>(o2::aod::track::TRD), (uint8_t)0x0) |
                                 ifnode((aod::track::tofChi2 >= 0.f) && (aod::track::tofExpMom > 0.f), static_cast<uint8_t>(o2::aod::track::TOF), (uint8_t)0x0));
 
+namespace v000
+{
+DECLARE_SOA_DYNAMIC_COLUMN(TrackTime, trackTime, //! Getter for TrackTime
+                           [](float trackTimeImpl) -> float {
```

Comment:
We should check for the AsymFlag. If set, we should return an invalid value. Is Tracktime always positive? Then -1, otherwise something large